docs/components/input-controller/webgamepad.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Configure a webgamepad"
+title: "Configure a Webgamepad"
```

Comment:
Nit: Normally I'd suggest Web Gamepad, as generic adj-noun in non-proper usage, however we specifically require that users use the name `WebGamepad` (incl. capitalization) per the copy below. Since this isn't a brand name, or product name, I'd therefore recommend one of the following. Unsure which is more appropriate!!

- `"Configure a Web Gamepad"` (to match general title case nature of this PR)
- `"Configure a WebGamepad"` (to match exactly required user-submitted string to get it working)

Reply:
```suggestion
title: "Configure a Web Gamepad"
```
Yeah...I was wondering about this myself. Okay will split it out.